src/import/sst_service.rs

```diff
@@ -714,3 +934,22 @@ fn make_request_header(mut context: Context) -> RaftRequestHeader {
     header.set_region_epoch(context.take_region_epoch());
     header
 }
+
+fn make_request(reqs: &mut HashMap<Vec<u8>, (Request, u64)>, context: Context) -> RaftCmdRequest {
+    let mut cmd = RaftCmdRequest::default();
+    let mut header = make_request_header(context);
+    // Set the UUID of header to prevent raftstore batching our requests.
```

Comment:
Where does the uuid prevent batching?

Reply:
Here, it seems raftstore would compare the header of requests, and would never batch two requests with different header.

<img width="890" alt="image" src="https://user-images.githubusercontent.com/36239017/167241320-0194f9e5-deaa-4d93-991d-97a7259ac327.png">